src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -87,6 +118,7 @@ private static void OpenConnection(string connString)
         {
             using SqlConnection connection = new(connString);
             connection.Open();
+            Assert.Equal(System.Data.ConnectionState.Open, connection.State);
```

Comment:
Can we instead use `try/catch` block to get a better exception handling of the thrown exception?

Reply:
There is no exception expected here.

Reply:
Correct me if I am wrong, if the assertion fails which means the connection state is not open, therefor an exception will be thrown on running the command.

Reply:
Yep - that's a test failure, so assert will fail.